setup.py

```diff
@@ -29,5 +29,5 @@ def install(package):
     packages=setuptools.find_packages(),
     test_suite="tangelo",
     setup_requires=['h5py'],
-    install_requires=['h5py', 'bitarray', 'openfermion', 'openfermionpyscf']
+    install_requires=['h5py', 'bitarray', 'openfermion', 'openfermionpyscf', 'pyqsp']
```

Comment:
I wonder if we should require this in the install, considering its just a single feature, maintained by a third party. These other requirements here are more important or closer to python core.

Maybe we should have a try / error block that tells the user "this feature requires pyqsp, please install the pyqsp python package".